datumaro/plugins/kitti_format/importer.py

```diff
@@ -75,3 +75,11 @@ def find_sources(path):
                 subsets.setdefault(subset_name, {})[task] = path
 
         return subsets
+
+class KittiDetectionImporter(KittiImporter):
+    _TASK = KittiTask.detection
+    _TASKS = { _TASK: KittiImporter._TASKS[_TASK] }
```

Comment:
This variable is not used in `find_sources()`

Reply:
Updated